app/models/ansible_playbooks.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Simple model to store basic info about the Ansible role
+class AnsiblePlaybook < ApplicationRecord
```

Comment:
Why do we need a new model for playbook if we are converting playbooks to job templates?